Delfin driver matrix support template

Shruthi-1MN opened the pull request:
<!--  Thanks for sending a pull request!  Here are some tips for you:

1. Please give clear description and fill all the needed fields in the PR template below
2. Provide all the test report and results for the PR. It is mandatory. Otherwise, 
   your PR may get rejected without any review/discussion
3. If the PR is incomplete/in progress, please add [WIP] at the beginning of the PR title.
4. Provide the link to the issue and other relevant files related to the PR
-->

**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespace from that line:
>
> /kind new feature
> /kind bug fix
> /kind cleanup
> /kind revert change
> /kind design
> /kind documentation
> /kind enhancement

**What this PR does / why we need it**:
This is PR, delfin driver support matrix template
**Which issue(s) this PR fixes**:
<!--
*Please provide the issues number or link.
Usage: `Fixes #<issue number>`, or `Fixes (paste link of issue)`.
-->
Fixes #

**Test Report Added?**:
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespace from that line:
> /kind TESTED
> /kind NOT-TESTED

**Test Report**:
<!--
*Please provide the test report link (public accessible, screen shot or copy paste the test report, 
or add the testing details.
-->

**Special notes for your reviewer**:

anvithks commented:
@Shruthi-1MN Please update the PR description with appropriate details.

anvithks commented on content/guides/user-guides/delfin/add-driver-support/_index.md:
```diff
@@ -0,0 +1,77 @@
+tags: ["user guide", "delfin", "drivers"]
+---
+Find below steps to add new delfin driver support matrix
+1. Clone the document repository (https://github.com/sodafoundation/documentation) 
```
document -> documentation
please correct the link syntax for repository.

anvithks commented on content/guides/user-guides/delfin/driver-matrix-support/_index.md:
```diff
@@ -0,0 +1,16 @@
+
+{{%children style="h3" description="true" %}} 
+
+If you want to add new delfin driver support matrix, find the link [here](https://docs.sodafoundation.io/guides/user-guides/delfin/add-driver-support/) 
```
Use relative path names.
https://docs.sodafoundation.io/guides/user-guides/delfin/add-driver-support -> /guides/user-guides/delfin/driver-matrix-support/

Shruthi-1MN replied:
Done

anvithks commented on content/guides/user-guides/delfin/driver-matrix-support/dell-emc-vmax/_index.md:
```diff
@@ -0,0 +1,54 @@
+
+### Model and Versions Supported
+
+|Models   | Firmware Versions   |
```
Please populate the content for the Dell EMC Vmax driver support.

Shruthi-1MN replied:
NA, it is just a template

anvithks replied:
This is not the template. this document will hold the values for Dell EMC VMAX driver.

Shruthi-1MN replied:
As per our discussion #144 has created to fill all the data

anvithks commented on content/guides/user-guides/delfin/driver-matrix-support/huawei-ocenstor/_index.md:
```diff
@@ -0,0 +1,54 @@
+
+### Model and Versions Supported
+
+|Models   | Firmware Versions   |
```
Please populate the content for the Huawei OceanStor driver support.

anvithks replied:
This file should contain the values for Huawei Oceanstor driver.

Shruthi-1MN replied:
#144 has been created to add data

anvithks reviewed (changes requested):
@Shruthi-1MN the Dell EMC and Huawei OceanStor driver details are empty. Please update the same.

NajmudheenCT reviewed (changes requested):
Can you please add screenshots of the pages here. un able to visualize the pages

anvithks commented:
> Can you please add screenshots of the pages here. un able to visualize the pages

![image](https://user-images.githubusercontent.com/19162717/106745098-a7916680-6646-11eb-81ba-034e5166869c.png)


@NajmudheenCT You can check out the deploy preview. It is generated as part of the All checks. Click on the [Details](https://deploy-preview-140--soda-docs.netlify.app/) link.

NajmudheenCT commented:
> > Can you please add screenshots of the pages here. un able to visualize the pages
> 
> ![image](https://user-images.githubusercontent.com/19162717/106745098-a7916680-6646-11eb-81ba-034e5166869c.png)
> 
> @NajmudheenCT You can check out the deploy preview. It is generated as part of the All checks. Click on the [Details](https://deploy-preview-140--soda-docs.netlify.app/) link.

Thanks, this helps

NajmudheenCT commented on content/guides/user-guides/delfin/driver-matrix-support/_index.md:
```diff
@@ -0,0 +1,16 @@
+
+{{%children style="h3" description="true" %}} 
+
+If you want to add new delfin driver support matrix, find the link [here](https://docs.sodafoundation.io/guides/user-guides/delfin/add-driver-support/) 
```
This is a broken link

Shruthi-1MN replied:
added relative link, it will work now

NajmudheenCT commented on content/guides/user-guides/delfin/driver-matrix-support/dell-emc-vmax/_index.md:
```diff
@@ -0,0 +1,54 @@
+|         |                     |
+|         |                     |
+
+### Access information for register storage
```
*for registering storage

Shruthi-1MN replied:
done

NajmudheenCT commented on content/guides/user-guides/delfin/driver-matrix-support/dell-emc-vmax/_index.md:
```diff
@@ -0,0 +1,54 @@
+
+### Access information for register storage
+
+|Access Type   | Attributes   | Description |
```
use mutiple rows in attributes  and descrition column for column for single access type , some thing as below


![image](https://user-images.githubusercontent.com/45681499/106747864-73b84000-664a-11eb-9dc5-81adaf531c26.png)

Shruthi-1MN replied:
Raised a issue #140

anvithks replied:
This should be fixed after #142 is merged. 
@kumarashit @skdwriting Please merge #142 so that HTML support is enabled.

@Shruthi-1MN after #142 is merged you will be able to use the HTML table tags.

NajmudheenCT commented on content/guides/user-guides/delfin/driver-matrix-support/dell-emc-vmax/_index.md:
```diff
@@ -0,0 +1,54 @@
+
+### Performance registration information
+
+| Resource | Interval | Historic collection available | Additional information |
```
Default Interval

Shruthi-1MN replied:
done

NajmudheenCT commented on content/guides/user-guides/delfin/driver-matrix-support/dell-emc-vmax/_index.md:
```diff
@@ -0,0 +1,54 @@
+
+### Resource data matrix
+
+| Category/Feature | Attribute | Status | Additional information |
```
change Category/Feature to -> resource

Shruthi-1MN replied:
done

NajmudheenCT commented on content/guides/user-guides/delfin/driver-matrix-support/dell-emc-vmax/_index.md:
```diff
@@ -0,0 +1,54 @@
+
+### Performance metric data
+
+| Category | Metric | Status | Additional information |
```
Category -> resource

Shruthi-1MN replied:
done

NajmudheenCT commented on content/guides/user-guides/delfin/driver-matrix-support/dell-emc-vmax/_index.md:
```diff
@@ -0,0 +1,54 @@
+| Category | Metric | Status | Additional information |
+|---------|----------|--------|-----------------------|
+|         | Capacity | Implemented |                  |
+| Pool    | IO       | Implemented |                  | 
```
remove this example

Shruthi-1MN replied:
done

NajmudheenCT commented on content/guides/user-guides/delfin/driver-matrix-support/_index.md:
```diff
@@ -0,0 +1,16 @@
+
+{{%children style="h3" description="true" %}} 
+
+If you want to add new delfin driver support matrix, find the link [here](https://docs.sodafoundation.io/guides/user-guides/delfin/add-driver-support/) 
```
Please add index to all supported drivers.. use the same template for all

Shruthi-1MN replied:
done

anvithks commented:
@Shruthi-1MN please avoid force pushing commits. It will overwrite previous comments and discussions.
Please push commits to the PR with out force push.

anvithks commented on content/guides/user-guides/delfin/driver-matrix-support/hp-3par-storeserv/_index.md:
```diff
@@ -0,0 +1,54 @@
+
+### Model and versions supported
+
+ |Models   | Firmware Versions   |
```
There seems to be some issue with the rendered table.
Please check the table.
![image](https://user-images.githubusercontent.com/19162717/107004599-89497900-67b4-11eb-8c6b-5f68a137a935.png)

Shruthi-1MN replied:
Rendering on my machine looks fine, PFA
![image](https://user-images.githubusercontent.com/48908765/107031105-acd1eb00-67d7-11eb-8556-89e942280429.png)

anvithks reviewed (changes requested):
@Shruthi-1MN  
Please populate the values for the different driver support matrix tables.
If these are supposed to be placeholders, then raise issues for the each table to be filled and these can be updated later.

NajmudheenCT reviewed (dismissed):
LGTM

kumarashit commented:
Few questions:
a) Can we not have a grouping at vendor level and then the model
   Dell EMC
                 VMAX
                 VNX
                 Unity
b) There is "description" for some and "additional information" for others. Are they mutually exclusive? Should not there be description for every attribute?
c) How do we maintain the model series? Ex: EMC VNX5200, EMC VNX 5600, EMC VNX7x, EMC VNX 8x?

anvithks commented:
> Few questions:
> a) Can we not have a grouping at vendor level and then the model
> Dell EMC
> VMAX
> VNX
> Unity
> b) There is "description" for some and "additional information" for others. Are they mutually exclusive? Should not there be description for every attribute?
> c) How do we maintain the model series? Ex: EMC VNX5200, EMC VNX 5600, EMC VNX7x, EMC VNX 8x?

@kumarashit @Shruthi-1MN 
We can have in page navigation like in this [page](https://docs.sodafoundation.io/guides/user-guides/delfin/dashboard/)
One page for each vendor and a section for each model. In page links to each section start.

Shruthi-1MN commented:
> Few questions:
> a) Can we not have a grouping at vendor level and then the model
> Dell EMC
> VMAX
> VNX
> Unity
> b) There is "description" for some and "additional information" for others. Are they mutually exclusive? Should not there be description for every attribute?
> c) How do we maintain the model series? Ex: EMC VNX5200, EMC VNX 5600, EMC VNX7x, EMC VNX 8x?

As per my research and understanding from OCI and stor2rrd document few description is for metrics, some places additional information is for resource information example if it is a protocol/firmware which they support and its dependency information

anvithks commented on content/support-matrix/delfin/hp/_index.md:
```diff
@@ -0,0 +1,10 @@
+---
+title: HP
```
HP -> HPE

Shruthi-1MN replied:
done

anvithks commented on content/support-matrix/delfin/hp/_index.md:
```diff
@@ -0,0 +1,10 @@
+---
+title: HP
+description: "All supported matrix data for HP Models can be found here"
```
Same as above HP -> HPE

Shruthi-1MN replied:
done

anvithks commented on content/support-matrix/_index.md:
```diff
@@ -0,0 +1,10 @@
+---
+title: Support Matrix
+description: "All supported matrix data for SODA can be found here"
```
All supported matrix data for SODA? 
This statement does not give the intended meaning.

Shruthi-1MN replied:
corrected the statements

anvithks commented on content/support-matrix/delfin/_index.md:
```diff
@@ -0,0 +1,12 @@
+tags: ["support matrix", "delfin", "drivers"] 
+---
+
+All support matrix data documents of Delfin drivers can be found here.
```
Will any documents will be added? If not then just data or tables.

Shruthi-1MN replied:
Changed the statement

anvithks commented on content/support-matrix/delfin/dell-emc/_index.md:
```diff
@@ -0,0 +1,10 @@
+---
+title: Dell Emc
```
Dell Emc -> Dell EMC

Shruthi-1MN replied:
done

anvithks commented on content/support-matrix/delfin/dell-emc/vmax/_index.md:
```diff
@@ -0,0 +1,448 @@
+     </tbody>
+ </table>
+ 
+ ### Access information to register storage
```
This does not appear as a heading in the deploy preview.
Could you please check again.

anvithks replied:
This is a screenshot from the deploy preview.
![image](https://user-images.githubusercontent.com/19162717/108018003-79dff080-703c-11eb-9cfa-0696801b8582.png)

Shruthi-1MN replied:
corrected

anvithks commented on content/support-matrix/delfin/dell-emc/vmax/_index.md:
```diff
@@ -0,0 +1,448 @@
+         </tr>
+         <tr>
+            <td>free_capacity</td>
+            <td></td>
```
what do each of the supported attributes mean?
There is no information provided to the user.

NajmudheenCT replied:
IMO @Shruthi-1MN We can provide description and other additional info in 'Additional info' tab. like used capacity in GB

sushanthakumar replied:
True, we can keep it as "Description" which can always contain fill some useful info (meaning, or unit, special note etc)

Shruthi-1MN replied:
done

anvithks reviewed (changes requested):
Attribute descriptions and values are not clear from the tables.
What are the values that some of the attributes take and what would be the best way to show it to the user?

NajmudheenCT commented on content/support-matrix/delfin/dell-emc/vmax/_index.md:
```diff
@@ -0,0 +1,448 @@
+             <td></td>
+         </tr>
+         <tr>
+            <td>native_storage_pool_id</td>
```
Give user readable filed name , like native_storage_pool_id can be named as 'storage pool id' follow this for all

Shruthi-1MN replied:
done

NajmudheenCT commented on content/support-matrix/delfin/dell-emc/vmax/_index.md:
```diff
@@ -0,0 +1,448 @@
+         <td></td>
+      </tr>
+      <tr>                                                                                                                                                                                                                                                                                                                                                                                                      ...
+           <td rowspan=18>Port</td>
```
Port, controller and disk information is not implemeted for VMAX. You need to check the driver implementaion file to see whether this interface is implemented for this driver 
![image](https://user-images.githubusercontent.com/45681499/108024745-24f7a680-704b-11eb-8395-368ebda8bdd8.png)

Shruthi-1MN replied:
done

NajmudheenCT commented on content/support-matrix/delfin/dell-emc/vmax/_index.md:
```diff
@@ -0,0 +1,448 @@
+ 
+ ### Resource data matrix
+ 
+ <table>
```
Device leval attributes are missing

Shruthi-1MN replied:
done

NajmudheenCT commented on content/support-matrix/delfin/dell-emc/vmax/_index.md:
```diff
@@ -0,0 +1,448 @@
+     </thead>
+     <tbody>
+         <tr>
+            <td>SYMAPI_AEVENT2_UID_EVT_RESTARTED</td>
```
This informations are wrong.. alert fileds means the fields available in delfin for VMAX alerts . you can probably mention the curresponding VMAX event field if applicable check with @sushanthakumar for more details 
![image](https://user-images.githubusercontent.com/45681499/108025047-c41c9e00-704b-11eb-9814-dcd7603cb13c.png)

Shruthi-1MN replied:
done

skdwriting commented:
1. Delfin Support Matrix : Currently we added driver support matrix. There can be other feature support matrix as well right? Is it not better to have a sub menu for Devices -> then the respective devices? Please check.
2. Should we call Support Matrix to Feature Support Matrix ?! (minor)
3. "ADD NEW DRIVER SUPPORT MATRIX" -- this is out of place!
4. The template does not look consistent across the drivers at the moment (including the format). Ex: Dell and HP
5. Should we give an introduction document for "How to read the support matrix"? we can give the details of how they are arranged and how it can be read easily.
6. Overall attribute description is missing
7. Additional Information -- this field, mostly will be blank?! Currently we waste a lot of view space for the same!
8. Data Type and Units could be added?
9. Should we give parameter mapping between delfin and the device in case they are different (name etc)
10. Can we have support matrix table as well, including the vendor and models with links to respective pages? Easy to traverse than menu. Because the supported devices are not visible currently in our menu. This index can provide an overall introduction as I mentioned in the point 5 above?

Shruthi-1MN commented on content/support-matrix/delfin/add-driver-support/_index.md:
```diff
@@ -0,0 +1,226 @@
+    <thead>
+        <tr>
+            <th>Models</th>
+            <th>Firmware Versions</th>
```
done

Shruthi-1MN commented on content/support-matrix/delfin/dell-emc/vmax/_index.md:
```diff
@@ -0,0 +1,448 @@
+     </thead>
+     <tbody>
+         <tr>
+            <td>VMAX 100K, VMAX 200K,VMAX 400K</td>
```
done

Shruthi-1MN commented on content/support-matrix/delfin/dell-emc/vmax/_index.md:
```diff
@@ -0,0 +1,448 @@
+     </tbody>
+ </table>
+ 
+ ### Access information to register storage
```
corrected

Shruthi-1MN commented on content/support-matrix/delfin/dell-emc/vmax/_index.md:
```diff
@@ -0,0 +1,448 @@
+     <tbody>
+         <tr>
+            <td rowspan="4">REST</td>
+            <td>host</td>
```
done

Shruthi-1MN commented on content/support-matrix/delfin/dell-emc/vmax/_index.md:
```diff
@@ -0,0 +1,448 @@
+     </thead>
+     <tbody>
+         <tr>
+            <td>Unisphere</td>
```
done

Shruthi-1MN commented:
>     1. Delfin Support Matrix : Currently we added driver support matrix. There can be other feature support matrix as well right? Is it not better to have a sub menu for Devices -> then the respective devices? Please check. 
>     2. Should we call Support Matrix to Feature Support Matrix ?! (minor)
Ans for 1 and 2. As the main menu is Support Matrix under that sub menu is delfin, so on parent menu we can any other support data as well ex: feature support matric
>     3. "ADD NEW DRIVER SUPPORT MATRIX" -- this is out of place!
It is within Delfin, felt better place let me know if you have better place to keep.
>     4. The template does not look consistent across the drivers at the moment (including the format). Ex: Dell and HP
Not clear, i copied the same template across all the drivers
>     5. Should we give an introduction document for "How to read the support matrix"? we can give the details of how they are arranged and how it can be read easily.
Added
>     6. Overall attribute description is missing
Added
>     7. Additional Information -- this field, mostly will be blank?! Currently we waste a lot of view space for the same!
Added, filled as much as possible
>     8. Data Type and Units could be added?
Added wherever is needed
>     9. Should we give parameter mapping between delfin and the device in case they are different (name etc)
Will add whenever it is required
>     10. Can we have support matrix table as well, including the vendor and models with links to respective pages? Easy to traverse than menu. Because the supported devices are not visible currently in our menu. This index can provide an overall introduction as I mentioned in the point 5 above?
I have added introduction at the beginning of delfin index page

NajmudheenCT commented on content/support-matrix/_index.md:
```diff
@@ -0,0 +1,12 @@
+---
+
+This section provides information about the support matrix data for SODA. 
+Please find below delfin project which has different vendor storage devices supported data matrix.
```
No need to mention delfin here.. this is common page holds index of all project support matrix. delfin is just one of them

NajmudheenCT commented on content/support-matrix/delfin/dell-emc/vmax/_index.md:
```diff
@@ -0,0 +1,363 @@
+
+### Supported alert source registration information
+
+<table>
```
Alert table format need changes.. Discuss with @sushanthakumar

Shruthi-1MN replied:
Done

NajmudheenCT commented on content/support-matrix/delfin/dell-emc/vmax/_index.md:
```diff
@@ -0,0 +1,363 @@
+      </tr>
+      <tr>
+         <td>Total Capacity</td>
+         <td>It is measured in terms of Tera byte</td>
```
All units are in bytes

Shruthi-1MN replied:
done

NajmudheenCT commented on content/support-matrix/delfin/dell-emc/vmax/_index.md:
```diff
@@ -0,0 +1,363 @@
+      </tr>
+      <tr>
+         <td>Subscribed Capacity</td>
+         <td>Refers to capacity allocated to device/host</td>
```
Not allocated.. it is the total amount of capacity subscribed

Shruthi-1MN replied:
Done

NajmudheenCT commented on content/support-matrix/delfin/dell-emc/vmax/_index.md:
```diff
@@ -0,0 +1,363 @@
+      </tr>
+      <tr>
+         <td>Type</td>
+         <td>Logical Type. Ex:Common LUN, Thin LUN</td>
```
Thick or Thin

Shruthi-1MN replied:
Done

NajmudheenCT commented on content/support-matrix/delfin/dell-emc/vmax/_index.md:
```diff
@@ -0,0 +1,363 @@
+      </tr>
+      <tr>
+         <td>Storage Pool Id</td>
+         <td>Unique id to represent which storage pool it belongs to</td>
```
It is ' SRP name '

Shruthi-1MN replied:
done

NajmudheenCT commented on content/support-matrix/delfin/add-driver-support/_index.md:
```diff
@@ -0,0 +1,227 @@
+tags: ["support matrix", "delfin", "drivers"]
+---
+
+Find below steps to add new delfin driver support matrix
```
Give link to sample template in the begining

Shruthi-1MN replied:
Added sample template in same page as per our discussion

NajmudheenCT commented on content/support-matrix/delfin/add-driver-support/_index.md:
````diff
@@ -0,0 +1,227 @@
+
+```
+10. Raise a PR to the documentation repository(https://github.com/sodafoundation/documentation.git)
+11. Please find sample data filled [template](/support-matrix/delfin/add-driver-support/_index.files/template.odt)
````
Give template as a md file.. it should be opened when clicking the link..

Shruthi-1MN replied:
same as above

NajmudheenCT commented on content/support-matrix/delfin/add-driver-support/_index.md:
````diff
@@ -0,0 +1,227 @@
+
+```
+10. Raise a PR to the documentation repository(https://github.com/sodafoundation/documentation.git)
+11. Please find sample data filled [template](/support-matrix/delfin/add-driver-support/_index.files/template.odt)
````
Some comments in the provided template file
1. No mentions of vendor here .
2. Treat it like support matrix for a 'Sample driver ' , where 'sample' is the vendor name
3. alert source table format to be changed!

Shruthi-1MN replied:
done

kumarashit commented on content/support-matrix/delfin/add-driver-support/_index.md:
```diff
@@ -0,0 +1,227 @@
+Find below steps to add new delfin driver support matrix
+1. Clone the documentation repository (https://github.com/sodafoundation/documentation.git) 
+2. Navigate to the folder structure content/support-matrix/delfin
+3. Create a new folder and follow the naming format `vendor`. For example `dell-emc`. 
```
This line is not very clear. esp. "naming format `vendor`"

Shruthi-1MN replied:
changed. done.

kumarashit commented on content/support-matrix/delfin/add-driver-support/_index.md:
````diff
@@ -0,0 +1,227 @@
+```
+---
+title: Storwize
+description: "All supported matrix data for IBM Models can be found here"
````
This description seem incorrect. Here we are already talking about IBM Storwize model. Isn't it?

Shruthi-1MN replied:
changed

kumarashit commented on content/support-matrix/delfin/add-driver-support/_index.md:
```diff
@@ -0,0 +1,227 @@
+disableToc: false
+---
+
+All supported matrix data for IBM Models which are supported by Delfin Project of SODA can be found here
```
This is also unaligned statement

Shruthi-1MN replied:
changed

kumarashit commented on content/support-matrix/delfin/add-driver-support/_index.md:
```diff
@@ -0,0 +1,227 @@
+Step 3, 4 and 5 are not required if there is already vendor folder name exists in documentation repository
+{{% /notice %}}
+
+6. Under vendor folder name, create a new folder with `model` name. For example `VNX`. 
```
nit: You started with IBM example for template so don't put VNX as example, rather put storwize

Shruthi-1MN replied:
changed with same driver name

kumarashit commented on content/support-matrix/delfin/dell-emc/_index.md:
```diff
@@ -0,0 +1,10 @@
+---
```
Why are we putting these template? Above file you have already provided the steps for creating matrix and given template. Let the driver writer/document owner create these files

Shruthi-1MN replied:
Later those will be filled. Issue has been raised for the same #144

kumarashit commented on content/support-matrix/delfin/dell-emc/vmax/_index.md:
```diff
@@ -0,0 +1,363 @@
+         <td>Device Location, is name-value pair having component type and component name</td>
+      </tr>
+      <tr>
+         <td>Total Capacity</td>
```
What exactly is Total capacity? Capacity of all the Disks or capacity of something else

Shruthi-1MN replied:
System capacity

NajmudheenCT replied:
Total Usable capacity

sushanthakumar commented on content/support-matrix/delfin/dell-emc/vmax/_index.md:
```diff
@@ -0,0 +1,363 @@
+      </tr>
+      <tr>
+         <td>Occur time</td>
+         <td>Alert trap created time</td>
```
Time at which alert is generated from device in epoch format

Shruthi-1MN replied:
done

sushanthakumar commented on content/support-matrix/delfin/dell-emc/vmax/_index.md:
```diff
@@ -0,0 +1,363 @@
+      </tr>
+      <tr>
+         <td>Location</td>
+         <td>Detailed info about the tracing the alerting device such as slot, rack etc.</td>
```
Detailed info about the tracing the alerting device such as array id, component name, component type

Shruthi-1MN replied:
done

sushanthakumar commented on content/support-matrix/delfin/dell-emc/vmax/_index.md:
```diff
@@ -0,0 +1,363 @@
+      </tr>
+      <tr>
+         <td>Category</td>
+         <td>Type of the reported alarm notification. Ex: New, Fixed, Removed etc.</td>
```
Category of the reported alarm notification. Ex: Fault, Event or Recovery

Shruthi-1MN replied:
done

sushanthakumar commented on content/support-matrix/delfin/dell-emc/vmax/_index.md:
```diff
@@ -0,0 +1,363 @@
+      </tr>
+      <tr>
+         <td>Sequence number</td>
+         <td>It maps to the trap</td>
```
Unique identification for the alert instance

Shruthi-1MN replied:
done

sushanthakumar commented on content/support-matrix/delfin/dell-emc/vmax/_index.md:
```diff
@@ -0,0 +1,363 @@
+      </tr>
+      <tr>
+         <td>Recovery advice</td>
+         <td>Recovery AdviceEx: SNMP authentication parameters are invalid</td>
```
This field does not contain any info. However description field may contain some recovery advice in some cases

Shruthi-1MN replied:
done

sushanthakumar commented on content/support-matrix/delfin/dell-emc/vmax/_index.md:
```diff
@@ -0,0 +1,363 @@
+      </tr>
+      <tr>
+         <td>Resource type</td>
+         <td>Resource type for alert</td>
```
Resource type for alert which can be storage subsystem or some other subsystems

Shruthi-1MN replied:
done

sushanthakumar commented on content/support-matrix/delfin/huawei/oceanstor/_index.md:
```diff
@@ -0,0 +1,517 @@
+      </tr>
+      <tr>
+         <td>Category</td>
+         <td>Type of the reported alarm notification. Ex: Fault, Event, Recovery</td>
```
Category of the reported alarm notification. Ex: Fault, Event, Recovery

Shruthi-1MN replied:
done

sushanthakumar commented on content/support-matrix/delfin/huawei/oceanstor/_index.md:
```diff
@@ -0,0 +1,517 @@
+      </tr>
+      <tr>
+         <td>Type</td>
+         <td>Alarm clearance type Ex:Communication Alarm, ProcessingErrorsAlarm</td>
```
Type of the alert generated such as CommunicationsAlarm, EquipmentAlarm etc.

Shruthi-1MN replied:
done

sushanthakumar commented on content/support-matrix/delfin/huawei/oceanstor/_index.md:
```diff
@@ -0,0 +1,517 @@
+      </tr>
+      <tr>
+         <td>Occur time</td>
+         <td>UTC time when the alarm is generated (in milliseconds)</td>
```
Time at which alert is generated from device. Specified in epoch format

Shruthi-1MN replied:
done

sushanthakumar commented on content/support-matrix/delfin/huawei/oceanstor/_index.md:
```diff
@@ -0,0 +1,517 @@
+         <td>Some suggestion for handling the given alert</td>
+      </tr>
+      <tr>
+         <td>Recovery type</td>
```
Resource type

Shruthi-1MN replied:
done

sushanthakumar commented on content/support-matrix/delfin/huawei/oceanstor/_index.md:
```diff
@@ -0,0 +1,517 @@
+      </tr>
+      <tr>
+         <td>Recovery type</td>
+         <td>Resource type of device/source generating alert</td>
```
Resource type of device/source generating alert. This is always set to "Storage".

Shruthi-1MN replied:
done

sushanthakumar commented on content/support-matrix/delfin/huawei/oceanstor/_index.md:
```diff
@@ -0,0 +1,517 @@
+      </tr>
+      <tr>
+         <td>Location</td>
+         <td>Detailed info about the tracing the alerting device such as slot, rack, etc.</td>
```
Location of an alarm. Specified in name/value format

Shruthi-1MN replied:
done

sushanthakumar commented on content/support-matrix/delfin/add-driver-support/_index.md:
```diff
@@ -0,0 +1,227 @@
+    </tbody>
+</table>
+
+### Supported alert source registration information
```
Alert management information

Shruthi-1MN replied:
done

sushanthakumar commented on content/support-matrix/delfin/dell-emc/unity/_index.md:
```diff
@@ -0,0 +1,179 @@
+
+### Supported alert source registration information
+
+<table>
```
![image](https://user-images.githubusercontent.com/30930862/108940245-001db780-7679-11eb-9fb4-36a16c911000.png)

sushanthakumar commented on content/support-matrix/delfin/dell-emc/vmax/_index.md:
```diff
@@ -0,0 +1,363 @@
+
+### Supported alert source registration information
+
+<table>
```
![image](https://user-images.githubusercontent.com/30930862/108940363-3a875480-7679-11eb-956b-b942b0670bb5.png)

Shruthi-1MN replied:
done

sushanthakumar commented on content/support-matrix/delfin/huawei/oceanstor/_index.md:
```diff
@@ -0,0 +1,517 @@
+
+### Supported alert source registration information
+
+<table>
```
![image](https://user-images.githubusercontent.com/30930862/108940618-b1245200-7679-11eb-8daa-b45994dd5b69.png)

Shruthi-1MN replied:
done

sushanthakumar commented on content/support-matrix/delfin/add-driver-support/_index.md:
```diff
@@ -0,0 +1,227 @@
+
+### Supported alert source registration information
+
+<table>
```
![image](https://user-images.githubusercontent.com/30930862/108940799-006a8280-767a-11eb-8087-59fe871dc32e.png)

Shruthi-1MN replied:
done

Shruthi-1MN commented on content/support-matrix/delfin/dell-emc/vmax/_index.md:
```diff
@@ -0,0 +1,363 @@
+      <tr>
+         <td rowspan="4">REST</td>
+         <td>Unisphere Host</td>
+         <td>Unishpere host name</td>
```
changed, done

Shruthi-1MN commented on content/support-matrix/delfin/dell-emc/vmax/_index.md:
```diff
@@ -0,0 +1,363 @@
+      </tr>
+      <tr>
+         <td>Storage Id</td>
+         <td>Id in the delfin system</td>
```
added proper description

Shruthi-1MN commented on content/support-matrix/delfin/dell-emc/vmax/_index.md:
```diff
@@ -0,0 +1,363 @@
+      </tr>
+      <tr>
+         <td>Total Capacity</td>
+         <td>It is measured in terms of Tera byte</td>
```
changed

Shruthi-1MN commented on content/support-matrix/delfin/dell-emc/vmax/_index.md:
```diff
@@ -0,0 +1,363 @@
+      </tr>
+      <tr>
+         <td>Type</td>
+         <td>Logical Type. Ex:Common LUN, Thin LUN</td>
```
corrected

Shruthi-1MN commented on content/support-matrix/delfin/dell-emc/vmax/_index.md:
```diff
@@ -0,0 +1,363 @@
+      </tr>
+      <tr>
+         <td>wwn</td>
+         <td>World wide name is to identify network storage devices</td>
```
Corrected

Shruthi-1MN commented on content/support-matrix/delfin/dell-emc/vmax/_index.md:
```diff
@@ -0,0 +1,363 @@
+      </tr>
+      <tr>
+         <td>Raw Capacity</td>
+         <td>It is measured in terms of Giga byte</td>
```
Physical Capacity, added in the description

anvithks reviewed (changes requested):
A few more comments.
Please check the rendered deploy preview to verify if the formatting is applied.

sushanthakumar commented:
LGTM

Shruthi-1MN commented on content/support-matrix/delfin/add-driver-support/_index.md:
```diff
@@ -0,0 +1,434 @@
+---
+
+Find below steps to add new delfin driver support matrix
+1. Clone the documentation repository (https://github.com/sodafoundation/documentation.git) 
```
done

anvithks replied:
Still looks like this in the [deploy preview](https://deploy-preview-140--soda-docs.netlify.app/support-matrix/delfin/add-driver-support/).
![image](https://user-images.githubusercontent.com/19162717/109102914-e0a58e00-774f-11eb-9812-7cc7c2f20ef9.png)

anvithks replied:
I believe this is because the numbered list expects `1.` and not `1)`
For a numbered list the number should be at the start of the line, no spaces and should be the number followed by a dot. 
At the end of a line to go to a new line you can either place two spaces or use `<br />`

anvithks replied:
I can see the same issue below as well @Shruthi-1MN 
![image](https://user-images.githubusercontent.com/19162717/109103253-8c4ede00-7750-11eb-8fa1-15ee1e6f19f9.png)

Shruthi-1MN commented on content/support-matrix/delfin/add-driver-support/_index.md:
```diff
@@ -0,0 +1,434 @@
+Please ensure you copy the entire frontmatter including both the `---`
+{{% /notice %}}
+
+9. To add the support matrix table content, copy lines from 57 to 432 from a file content/support-matrix/delfin/add-driver-support/_index.md 
```
Corrected the line

Shruthi-1MN commented on content/support-matrix/delfin/add-driver-support/_index.md:
```diff
@@ -0,0 +1,434 @@
+   </tbody>
+</table>
+
+12. Raise a PR to the documentation repository(https://github.com/sodafoundation/documentation.git)
```
yes removed

anvithks commented on content/support-matrix/delfin/add-driver-support/_index.md:
```diff
@@ -0,0 +1,434 @@
+         <td>Management software version</td>
+      </tr>
+      <tr>
+         <td></td>
```
Remove extra blank row. 
OR
**Question**: Will the Management software versions be a comma separated list?
If the blank row is for reference you can write something like:
|Model name|Management Software Versions|
|---------------------------|----------------------------------------------|
|< Enter model name here>|< Management software versions in comma separated list >|
|< Enter model name here>|< Management software versions in comma separated list >|

anvithks replied:
Never mind @Shruthi-1MN 
I saw [this](https://deploy-preview-140--soda-docs.netlify.app/support-matrix/delfin/dell-emc/vmax/) and realised it is the other way around.

![image](https://user-images.githubusercontent.com/19162717/109105468-fe74f200-7753-11eb-887c-d538d3544df4.png)

anvithks replied:
Please do remove the extra blank rows from the template.

Shruthi-1MN replied:
done

anvithks commented on content/support-matrix/delfin/add-driver-support/_index.md:
```diff
@@ -0,0 +1,434 @@
+<table>
+   <tbody>
+      <tr>
+          <th>SNMP trap processing</th> 
```
Why the `<th>` inside a row and why all the rows are spanned together?
![image](https://user-images.githubusercontent.com/19162717/109104334-cd93bd80-7751-11eb-8f9d-75f1b0ef82d4.png)

Shruthi-1MN replied:
It's default table style rendering has issue

anvithks replied:
Do we need a row header? We can just have a regular table as we have for the other information.

anvithks replied:
@Shruthi-1MN 
Instead of a `<th>` can you try using a regular `<td>` and put the content inside `<strong>This is bold text</strong>` For ex: <strong>This is bold text</strong>.  
```html
<table>
<tr>
<td><strong>Query Backend Alert</strong></td>
<td>Supported</td>
</tr>
</table>
```
<table>
<tr>
<td><strong>Query Backend Alert</strong></td>
<td>Supported</td>
</tr>
</table>

anvithks commented on content/support-matrix/delfin/add-driver-support/_index.md:
```diff
@@ -0,0 +1,434 @@
+
+### Performance registration information
+
+<table>
```
This table is empty and is not formatted correctly. Please check

Shruthi-1MN replied:
updated sample data

anvithks commented on content/support-matrix/delfin/add-driver-support/_index.md:
```diff
@@ -0,0 +1,434 @@
+   </tbody>
+</table>
+
+12. Raise a PR to the documentation repository
```
This is how the end appears in the deploy preview.
The `12.` is getting rendered as `1.` since it is the first item in a numbered list with correct syntax.
I think fixing all the above numbering will fix this. 
![image](https://user-images.githubusercontent.com/19162717/109104553-324f1800-7752-11eb-981c-3a1c3b187c3e.png)

Shruthi-1MN replied:
done

anvithks commented on content/support-matrix/delfin/add-driver-support/_index.md:
```diff
@@ -0,0 +1,434 @@
+ 
+### Performance metric data
+ 
+<table>
```
This table is empty and not formatted correctly. Please check if any additional information needs to be provided.

Shruthi-1MN replied:
addressed

anvithks commented on content/support-matrix/delfin/_index.md:
```diff
@@ -0,0 +1,32 @@
+Delfin project aims to overcome a challenge to manage heterogeneous storage devices on a single platform.
+Delfin has standardized common API services. 
+
+ 1) Registering the devices for the following operations
```
Please check the formatting.
Use `1.` instead of `1)`
You can refer to this [markdown guide by Github](https://guides.github.com/features/mastering-markdown/)
![image](https://user-images.githubusercontent.com/19162717/109105025-1ef07c80-7753-11eb-96f3-8a2b0c233cb9.png)

Shruthi-1MN replied:
thanks and changed as per the refernce

anvithks commented on content/support-matrix/delfin/dell-emc/vmax/_index.md:
```diff
@@ -0,0 +1,370 @@
+tags: ["support matrix", "delfin", "drivers", "dell emc"]
+---
+
+### Model and Management software versions supported
```
**Question**: Are we going to update this page with actual data? Except for the Model and Management software versions supported table the other tables all seem to be the same as the template. Are we leaving these to be filled by community members or driver authors?

anvithks replied:
@Shruthi-1MN @NajmudheenCT 
The individual driver support matrix files that are part of the PR are currently empty. How are we going ahead with this?

NajmudheenCT replied:
Yes anvit. We can remove that. @Shruthi-1MN Please remove other indexes , keep only ( Add new driver, VMAX and Oceanstoe , sections)

anvithks reviewed (changes requested):
@Shruthi-1MN @NajmudheenCT 
Some doubts about the supported drivers that we have added here. 
All the tables are empty. Are we placing these here as the first step and asking someone else to fill them in?

NajmudheenCT commented on content/support-matrix/delfin/add-driver-support/_index.md:
```diff
@@ -0,0 +1,434 @@
+      </tr>
+      <tr>
+         <td>Used Capacity</td>
+         <td>SRP usable used capacity(TB)</td>
```
SRP is not a common ter.. just say 'Storage pool Used capacity' unit is bytes

NajmudheenCT commented on content/support-matrix/delfin/add-driver-support/_index.md:
```diff
@@ -0,0 +1,434 @@
+      </tr>
+      <tr>
+         <td>Total Capacity</td>
+         <td>SRP(Storage resource pool) usable total capacity(TB)</td>
```
Remove SRP , it is only a VMAX term , in template we can say 'Storage Pool'

NajmudheenCT commented on content/support-matrix/delfin/add-driver-support/_index.md:
```diff
@@ -0,0 +1,434 @@
+      </tr>
+      <tr>
+         <td>Used Capacity</td>
+         <td>It is measured in TB</td>
```
Remove measured in TB , say ' Volume Used capacity'

NajmudheenCT commented on content/support-matrix/delfin/add-driver-support/_index.md:
```diff
@@ -0,0 +1,434 @@
+      </tr>
+      <tr>
+         <td>Free Capacity</td>
+         <td>It is measured in TB</td>
```
same as above

NajmudheenCT commented on content/support-matrix/delfin/add-driver-support/_index.md:
```diff
@@ -0,0 +1,434 @@
+      </tr>
+      <tr>
+         <td>Serial Number</td>
+         <td>It is VMAX array id</td>
```
remove VMAX

NajmudheenCT commented on content/support-matrix/delfin/add-driver-support/_index.md:
```diff
@@ -0,0 +1,434 @@
+      </tr>
+      <tr>
+         <td>Total Capacity</td>
+         <td>System capacity(TB)</td>
```
Total Usable capacity of the array

NajmudheenCT commented on content/support-matrix/delfin/add-driver-support/_index.md:
```diff
@@ -0,0 +1,434 @@
+      </tr>
+      <tr>
+         <td>Used Capacity</td>
+         <td>It is measured in TB</td>
```
Toatl Used capacity

NajmudheenCT commented on content/support-matrix/delfin/add-driver-support/_index.md:
```diff
@@ -0,0 +1,434 @@
+      </tr>
+      <tr>
+         <td>Free Capacity</td>
+         <td>It is measured in TB</td>
```
Total free capacity

NajmudheenCT commented on content/support-matrix/delfin/add-driver-support/_index.md:
```diff
@@ -0,0 +1,434 @@
+      </tr>
+      <tr>
+         <td>Raw Capacity</td>
+         <td>Physical capacity(GB)</td>
```
Total Raw Capacity ( sum of all disk capacity)

NajmudheenCT commented on content/support-matrix/delfin/add-driver-support/_index.md:
```diff
@@ -0,0 +1,434 @@
+      </tr>
+      <tr>
+         <td>Subscribed Capacity</td>
+         <td>Total amount of capacity subscribed(TB)</td>
```
unit is bytes everywhere

NajmudheenCT commented on content/support-matrix/delfin/add-driver-support/_index.md:
```diff
@@ -0,0 +1,434 @@
+ 
+### Performance metric data
+ 
+<table>
```
We can give some samples in the template. 
follow this 
![image](https://user-images.githubusercontent.com/45681499/109130262-86b8be80-7777-11eb-95e4-52483910a224.png)

NajmudheenCT commented on content/support-matrix/delfin/add-driver-support/_index.md:
```diff
@@ -0,0 +1,434 @@
+---
+
+Find below steps to add new delfin driver support matrix
+1) Clone the [documentation repository](https://github.com/sodafoundation/documentation.git)
```
Should we put template with sample data first and then the steps to add new support matric page? @anvithks What do you think?

anvithks replied:
Keep the sample data tables at the bottom of the page and give an in page link using the #id of the heading.

NajmudheenCT commented on content/support-matrix/delfin/dell-emc/vmax/_index.md:
```diff
@@ -0,0 +1,370 @@
+      </tr>
+      <tr>
+         <th>Clear backend alert</th>
+         <td>Supported</td>
```
Supported only for 'unipshere alert ' @sushanthakumar please confirm

sushanthakumar replied:
> Supported only for 'unipshere alert ' @sushanthakumar please confirm

Yes, query backend source is mentioned as unisphere which can be cleared, so we can mention clear as supported

NajmudheenCT commented on content/support-matrix/delfin/dell-emc/vmax/_index.md:
```diff
@@ -0,0 +1,370 @@
+      </tr>
+      <tr>
+         <td>Total Capacity</td>
+         <td>SRP(Storage resource pool) usable total capacity(TB)</td>
```
unit is bytes , everywhere

NajmudheenCT commented on content/support-matrix/delfin/dell-emc/vmax/_index.md:
```diff
@@ -0,0 +1,370 @@
+      </tr>
+      <tr>
+         <td>Free Capacity</td>
+         <td>Total-Used capacity(TB)</td>
```
Total used capacity of SRP

NajmudheenCT commented on content/support-matrix/delfin/dell-emc/vmax/_index.md:
```diff
@@ -0,0 +1,370 @@
+      </tr>
+      <tr>
+         <td>Used Capacity</td>
+         <td>It is measured in TB</td>
```
LUN Used Capacity

NajmudheenCT commented on content/support-matrix/delfin/dell-emc/vmax/_index.md:
```diff
@@ -0,0 +1,370 @@
+      </tr>
+      <tr>
+         <td>Free Capacity</td>
+         <td>It is measured in TB</td>
```
LUN Free capacity

NajmudheenCT commented on content/support-matrix/delfin/dell-emc/vmax/_index.md:
```diff
@@ -0,0 +1,370 @@
+      </tr>
+      <tr>
+         <td>WWN</td>
+         <td>Volume's WWN</td>
```
LUN wwn

NajmudheenCT commented on content/support-matrix/delfin/dell-emc/vmax/_index.md:
```diff
@@ -0,0 +1,370 @@
+      </tr>
+      <tr>
+         <td>Used Capacity</td>
+         <td>It is measured in TB</td>
```
Total Used Capacity

NajmudheenCT commented on content/support-matrix/delfin/dell-emc/vmax/_index.md:
```diff
@@ -0,0 +1,370 @@
+      </tr>
+      <tr>
+         <td>Free Capacity</td>
+         <td>It is measured in TB</td>
```
total Free capacity

NajmudheenCT commented on content/support-matrix/delfin/dell-emc/vmax/_index.md:
```diff
@@ -0,0 +1,370 @@
+      </tr>
+      <tr>
+         <td>Raw Capacity</td>
+         <td>Physical capacity(GB)</td>
```
Raw Capacity ( sum of all disks capacity)

NajmudheenCT commented on content/support-matrix/delfin/dell-emc/vmax/_index.md:
```diff
@@ -0,0 +1,370 @@
+      </tr>
+      <tr>
+         <td>Subscribed Capacity</td>
+         <td>Total amount of capacity subscribed(TB)</td>
```
unit is bytes

NajmudheenCT commented on content/support-matrix/delfin/add-driver-support/_index.md:
```diff
@@ -0,0 +1,424 @@
+         <td></td>
+      </tr>
+      <tr>
+         <td>Write Bandwidth</td>
```
These are not in resource column, Please follow the example I have given for other resource also..

Shruthi-1MN replied:
Added for other resources

NajmudheenCT commented on content/support-matrix/delfin/add-driver-support/_index.md:
```diff
@@ -0,0 +1,424 @@
+10. Fill appropriate driver data to support matrix tables.  
+11. Please find below sample driver support matrix data tables with sample data for reference to fill specific driver support matrix data.  
+12. Raise a PR to the documentation repository  
+# Sample Driver Name 
```
Give a heading here " Support matrix template "

Shruthi-1MN replied:
Update the header

sushanthakumar commented:
LGTM

NajmudheenCT commented on content/support-matrix/delfin/_index.md:
```diff
@@ -0,0 +1,32 @@
+tags: ["support matrix", "delfin", "drivers"] 
+---
+
+Delfin project aims to overcome a challenge to manage heterogeneous storage devices on a single platform.
```
This content needs to be modified . 
Please take below content and modify if required..

Delfin project aims to overcome a challenge to manage heterogeneous storage devices on a single platform. Delfin has standardized common API services.
Support matrix of each storage driver are grouped to their vendor name.
Each driver support matrix include below topics.
1.	Model and management software versions supported:-  List of storage models and required management software versions.
2.	Access information to register storage: Table illustrates what all access protocol supported and parameters required to register the storage device.
3.	Alert management information : Type of alert sourcing and clearing supported

4.	Performance registration information:  Table illustrates the resources and their default interval supported for performance collection.
5.	Resource data matrix:  Table illustrates the resources supported anlog with their available attributes.
6.	Alert data matrix : Table illustrates fields available  in the alert data 
7.	Performance metric data: Table illustrates performance metrics supported for each resource.

NajmudheenCT commented on content/support-matrix/delfin/add-driver-support/_index.md:
```diff
@@ -0,0 +1,494 @@
+         <td></td>
+      </tr>
+      <tr>
+         <td>Read throughput</td>
```
Give only 2 or 3 metrics in each resource.. Give description for all.. Do not leave blank

NajmudheenCT reviewed (approved):
LGTM

anvithks commented on content/support-matrix/delfin/_index.md:
```diff
@@ -0,0 +1,26 @@
+Delfin project aims to overcome a challenge to manage heterogeneous storage devices on a single platform.
+Delfin has standardized common API services. 
+
+Support matrix of each storage drivers are group to their vendor name.
```
Support matrix of each storage drivers are group to their vendor name. -> _Support matrix of storage drivers are grouped by their vendors_

Shruthi-1MN replied:
done

anvithks commented on content/support-matrix/delfin/_index.md:
```diff
@@ -0,0 +1,26 @@
+Delfin has standardized common API services. 
+
+Support matrix of each storage drivers are group to their vendor name.
+Each driver support matrix include below topics.
```
include -> includes

Shruthi-1MN replied:
done

anvithks commented on content/support-matrix/delfin/dell-emc/vmax/_index.md:
```diff
@@ -0,0 +1,354 @@
+      </tr>
+   </thead>
+   <tbody>
+      <tr>
```
@Shruthi-1MN Table is empty. remove if not available or applicable.

Shruthi-1MN replied:
@NajmudheenCT has to confirm

anvithks commented on content/support-matrix/delfin/dell-emc/vmax/_index.md:
```diff
@@ -0,0 +1,354 @@
+   </thead>
+   <tbody>
+      <tr>
+         <td rowspan=3></td>
```
Table is empty. Please remove if not available or applicable

Shruthi-1MN replied:
same as above

anvithks commented on content/support-matrix/delfin/huawei/oceanstor/_index.md:
```diff
@@ -0,0 +1,524 @@
+   </thead>
+   <tbody>
+      <tr>
+         <td>Storage Management Software</td>
```
This does not seem correct. Please check the values @Shruthi-1MN @NajmudheenCT

Shruthi-1MN replied:
It is correct

anvithks replied:
@Shruthi-1MN Under the `Models` column It says `Storage Management Software`. Shouldn't this be a list of Oceanstor models?

anvithks commented on content/support-matrix/delfin/huawei/oceanstor/_index.md:
```diff
@@ -0,0 +1,524 @@
+          <td>Supported</td> 
+      </tr>
+      <tr>
+         <th>Query backend alert</th>
```
Instead of a `<th>` can you try using a regular `<td>` and put the content inside `<strong>This is bold text</strong>` For ex: <strong>This is bold text</strong>.  
```html
<table>
<tr>
<td><strong>Query Backend Alert</strong></td>
<td>Supported</td>
</tr>
</table>
```
<table>
<tr>
<td><strong>Query Backend Alert</strong></td>
<td>Supported</td>
</tr>
</table>

Shruthi-1MN replied:
changed

anvithks commented on content/support-matrix/delfin/huawei/oceanstor/_index.md:
```diff
@@ -0,0 +1,524 @@
+      </tr>
+   </thead>
+   <tbody>
+      <tr>
```
Table is empty. Remove if not available or applicable.

Shruthi-1MN replied:
@NajmudheenCT has to confirm

anvithks commented on content/support-matrix/delfin/huawei/oceanstor/_index.md:
```diff
@@ -0,0 +1,524 @@
+         <th>Additional information</th>
+      </tr>
+   </thead>
+   <tbody>
```
Table is empty. Remove if not available or applicable.

Shruthi-1MN replied:
same as above

anvithks commented on content/support-matrix/delfin/add-driver-support/_index.md:
```diff
@@ -0,0 +1,441 @@
+{{% notice note %}}
+Step 3, 4 and 5 are not required if there is already vendor folder name exists in the documentation repository
+{{% /notice %}}  
+6. Under vendor folder name, create a new folder with `model` name. For example `VNX`.  
```
nit: `VNX` -> `vnx`
We use lowercase letters for folder names.

Shruthi-1MN replied:
done

anvithks commented on content/support-matrix/delfin/add-driver-support/_index.md:
```diff
@@ -0,0 +1,441 @@
+      </tr>
+      <tr>
+         <td>Storage Id</td>
+         <td>Id generated by delfin to identify the storage device</td>
```
nit: delfin -> Delfin

Shruthi-1MN replied:
done

anvithks commented on content/support-matrix/delfin/add-driver-support/_index.md:
```diff
@@ -0,0 +1,441 @@
+      </tr>
+      <tr>
+         <td>Total Capacity</td>
+         <td>total pool capacity(bytes)</td>
```
nit: total -> Total

Shruthi-1MN replied:
done

anvithks commented on content/support-matrix/delfin/add-driver-support/_index.md:
```diff
@@ -0,0 +1,441 @@
+      </tr>
+      <tr>
+         <td>Free Capacity</td>
+         <td>Total-Used capacity(TB)</td>
```
Total-Used -> Total used

Shruthi-1MN replied:
done

anvithks commented on content/support-matrix/delfin/add-driver-support/_index.md:
```diff
@@ -0,0 +1,441 @@
+      </tr>
+      <tr>
+         <td>Storage Id</td>
+         <td>Id generated by delfin to identify the storage device</td>
```
delfin -> Delfin

Shruthi-1MN replied:
done

anvithks reviewed (changes requested):
Some formatting issues and minor changes. @Shruthi-1MN I think this is finally ready.

anvithks reviewed (dismissed):
Overall LGTM.
@Shruthi-1MN Please check one comment in the Oceanstor driver file.
There are a few comments on some of the empty tables. @NajmudheenCT Please do take a look.

anvithks commented:
@NajmudheenCT Please review and approve again as your earlier review was rejected due to new commits.

NajmudheenCT commented:
> Overall LGTM.
> @Shruthi-1MN Please check one comment in the Oceanstor driver file.
> There are a few comments on some of the empty tables. @NajmudheenCT Please do take a look.

@Shruthi-1MN Let us remove the emty tables , We can add perf-collection related info after this release.

anvithks reviewed (approved):
LGTM.

NajmudheenCT reviewed (approved):
LGTM

sushanthakumar commented:
LGTM

anvithks commented:
@kumarashit @skdwriting @Shruthi-1MN this PR depends on #142 for enabling HTML tags in the docs website.
Please merge that PR before this one.
